tests/common/plugins/sanity_check/recover.py

```diff
@@ -123,3 +122,5 @@ def neighbor_vm_restore(duthost, nbrhosts, tbinfo):
             asn = nbrhosts[peer_device]['conf']['bgp']['asn']
             # restore BGP session
             nbr_host.no_shutdown_bgp(asn)
+            # start BGPd
+            nbr_host.start_bgpd()
```

Comment:
should we bring up the bgp session, or start bgpd process?

Reply:
Maybe the ordering should be start bgpd, then start bgp sessinos? I believe they are intended to fix 2 different issues?

Reply:
The two calls made here are:
already present: `no_shutdown_bgp`: this one issues `no shut router bgp` (BGP session bring up)
new addition here: `start_bgpd`: this issues `no agent Rib shutdown` (BGPd process bring up)

So I think this would cover both scenarios.

Reply:
does this comment affect if the bgpd is not down?

Reply:
No, if BGP is already running, it stays running.